etc/profile-m-z/makedeb.profile

```diff
@@ -0,0 +1,24 @@
+# Firejail profile for makedeb
+# Description: An utility to automate the building of Debian packages
```

Comment:
```suggestion
# Description: A utility to automate the building of Debian packages
```

Grammar; same as the following:

* <https://github.com/netblue30/firejail/pull/5299#discussion_r941730616>